components/script/dom/dommatrixreadonly.rs

```diff
@@ -699,6 +702,129 @@ impl DOMMatrixReadOnlyMethods for DOMMatrixReadOnly {
         create_buffer_source(cx, &self.matrix.borrow().to_array(), array.handle_mut())
             .expect("Converting matrix to float64 array should never fail")
     }
+
+    // https://drafts.fxtf.org/geometry/#dommatrixreadonly-stringification-behavior
```

Comment:
```suggestion
    /// <https://drafts.fxtf.org/geometry/#dommatrixreadonly-stringification-behavior>
```

Reply:
That method is in a trait impl, I don't think making this a doc comment changes anything?